### PR TITLE
DOCS: Auth0 JWT signature algorithm

### LINKED ISSUE #3124: Additional parameter in Auth0
Currently, I set up Auth0 IdP with Pomerium but my authentication with Auth0 wouldn't work. It took me several hours to figure out but once I switched JWT signature algorithm to RS256, it worked right away. I think it'd be helpful to add that to the documentation for clarity

image

### PR DESCRIPTION
**Pull request by alexfornuto**
## Summary

Documents the required JWT signature algorithm.

## Related issues

Fixes #3124 

## Checklist

- [x] reference any related issues
- [x] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review

### CODE REVIEW

File: docs/docs/identity-providers/auth0.md [hunk 1]
```diff
@@ -35,7 +35,9 @@ While we do our best to keep our documentation up to date, changes to third-part
    | Application Login URI        | [Authenticate Service URL] (e.g. `https://${authenticate_service_url}`)   |
    | Allowed Callback URLs        | Redirect URL (e.g. `https://${authenticate_service_url}/oauth2/callback`).|
 
-   Make sure to click **Save Changes** at the bottom of the page when you're done.
+1. Under **Advanced Settings** → **Oauth**, confirm that **JSON Web Token (JWT) Signature Algorithm** is set to "RS256".
```
Comment (travisgroth) [inline docs/docs/identity-providers/auth0.md:38]:
```suggestion
1. Under **Advanced Settings** → **auth0**, confirm that **JSON Web Token (JWT) Signature Algorithm** is set to "RS256".
```

**Review by travisgroth — CHANGES_REQUESTED** [review]

File: docs/docs/identity-providers/auth0.md (hunk 1 above)
Comment (alexfornuto) [reply docs/docs/identity-providers/auth0.md:38]: derp.

Comment (alexfornuto) [reply docs/docs/identity-providers/auth0.md:38]: @travisgroth that's a capital o, not a zero.

Comment (alexfornuto) [reply docs/docs/identity-providers/auth0.md:38]:
```suggestion
1. Under **Advanced Settings** → **OAuth**, confirm that **JSON Web Token (JWT) Signature Algorithm** is set to "RS256".
```

Comment (alexfornuto) [reply docs/docs/identity-providers/auth0.md:38]: image

Comment (travisgroth) [reply docs/docs/identity-providers/auth0.md:38]: 🤦 

Would suggest `oauth` or `OAuth` then

**Review by travisgroth — APPROVED** [review]

**Pull request merged**